### PR TITLE
fix: total-gas query fix

### PR DESCRIPTION
**Pull request by oren-lava**
# Description

Closes: #XXXX

I've added proper support for the `--height` flag for the `total-gas` query

---

## Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [ ] read the [contribution guide](https://github.com/lavanet/lava/blob/main/CONTRIBUTING.md)
* [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title, you can find examples of the prefixes below:
    
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] targeted the `main` branch
* [ ] provided a link to the relevant issue or specification
* [ ] reviewed "Files changed" and left comments if necessary
* [ ] included the necessary unit and integration tests
* [ ] updated the relevant documentation or specification, including comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] confirmed all CI checks have passed

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed all author checklist items have been addressed
* [ ] reviewed state machine logic, API design and naming, documentation is accurate, tests and test coverage

### CODE REVIEW

File: utils/cli/multisend.go [hunk 1]
```diff
@@ -366,7 +367,7 @@ func NewQueryTotalGasCmd() *cobra.Command {
 			authquerier := authtypes.NewQueryClient(clientCtx)
 
 			account := args[0]
-			chainid := args[1]
+			// chainid := args[1]
```
Comment (omerlavanet) [inline utils/cli/multisend.go:370]: need to fix the cli usage and explanation

File: utils/cli/multisend.go [hunk 2]
```diff
@@ -405,26 +406,48 @@ func NewQueryTotalGasCmd() *cobra.Command {
...
 				tx := getResponse(account, sequence)
 
-				if strings.Contains(tx.RawLog, chainid) && strings.Contains(tx.RawLog, "MsgRelayPayment") {
-					totalgas += tx.GasUsed
-				}
-				// Parse the time string
-				txtime, err = time.Parse(layout, tx.Timestamp)
-				if err != nil {
-					return err
+				if tx != nil {
```
Comment (omerlavanet) [inline utils/cli/multisend.go:427]: infinite loop on tx = nil

File: utils/cli/multisend.go [hunk 3]
```diff
@@ -405,26 +406,48 @@ func NewQueryTotalGasCmd() *cobra.Command {
...
+				now = res.Block.Header.Time
+			}
+
 			txtime := now
 			totalgas := int64(0)
+			numRelays := int64(0)
 			sequence, _ := getSequence(account)
 			layout := time.RFC3339
-			for now.Sub(txtime) < 24*time.Hour {
+			for now.Sub(txtime) < 12*time.Hour {
```
Comment (omerlavanet) [inline utils/cli/multisend.go:423]: update cli help to 12h

File: utils/cli/multisend.go (hunk 1 above)
Comment (omerlavanet) [inline utils/cli/multisend.go:370]: remove this comment

Comment (oren-lava) [reply utils/cli/multisend.go]: done. see https://github.com/lavanet/lava/commit/5b44b0fc38bc2ab755612c85c83be4c287d25e9d

File: utils/cli/multisend.go (hunk 2 above)
Comment (oren-lava) [reply utils/cli/multisend.go]: done. see https://github.com/lavanet/lava/commit/5b44b0fc38bc2ab755612c85c83be4c287d25e9d

File: utils/cli/multisend.go (hunk 3 above)
Comment (oren-lava) [reply utils/cli/multisend.go:422]: done. see https://github.com/lavanet/lava/commit/5b44b0fc38bc2ab755612c85c83be4c287d25e9d

File: utils/cli/multisend.go (hunk 1 above)
Comment (oren-lava) [reply utils/cli/multisend.go]: done. see https://github.com/lavanet/lava/commit/5b44b0fc38bc2ab755612c85c83be4c287d25e9d

**Review by omerlavanet — APPROVED** [review]

**Pull request merged**